### PR TITLE
Filter display

### PR DESCRIPTION
**Pull request by PierreGoe**

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -21,6 +21,10 @@
       rel="stylesheet"
       href="https://unpkg.com/react-leaflet-markercluster/dist/styles.min.css"
     />
+    <link
+      href="https://fonts.googleapis.com/icon?family=Material+Icons"
+      rel="stylesheet"
+    />
```
Comment (PierreGoe) [inline public/index.html]: empty line

**Review by LuigiMenez — DISMISSED** [review]
Ok for me

File: src/Components/Map/style.jsx [hunk 2]
```diff
@@ -52,6 +52,34 @@ const SMap = styled.div`
     width: 100vw;
     z-index: 1;
   }
+
```
Comment (Hacene010) [inline src/Components/Map/style.jsx]: Empty line !

File: src/Components/Map/style.jsx [hunk 3]
```diff
@@ -52,6 +52,34 @@ const SMap = styled.div`
...
+      margin: 2vh 3vw;
+      z-index: 10;
+      display: flex;
+      flex-direction: column-reverse;
+      align-items: center;
+      height: 100%;
+      justify-content: space-between;
+      margin: 0;
+    }
+
```
Comment (Hacene010) [inline src/Components/Map/style.jsx]: empty line !

File: src/style.jsx [hunk 4]
```diff
@@ -7,7 +7,7 @@ const GlobalStyle = createGlobalStyle`
     --color-primary: #28abb9;
     --color-secondary: #2d6187;
     --color-third: #a8dda8;
-    --font-family: 'Dekko', cursive;
+    --font-family: 'Dekko';
   }
```
Comment (Hacene010) [inline src/style.jsx]: empty line !

File: public/index.html (hunk 1 above)
Comment (Hacene010) [reply public/index.html]: Plz correct this !

**Review by Hacene010 — CHANGES_REQUESTED** [review]

**Review by LuigiMenez — APPROVED** [review]
Ok for me

**Review by Hacene010 — APPROVED** [review]

**Pull request merged**